srcStatic/LoggerDistributor.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Logger.h"
+#include <vector>
+
+
+class LoggerDistributor : public Logger
+{
+public:
+	LoggerDistributor(std::vector<Logger*> loggers);
```

Comment:
What do you think about adding a comment to the constructor explaining how ownership of the loggers is supposed to work? I think these are more reference than pointers.

Reply:
Probably a good idea to document (non-)ownership characteristics, and non-nullability. I had originally tried to use references, but found that created a problem for vectors.